source/backend/api/Services/PropertyService.cs

```diff
@@ -302,15 +305,7 @@ public bool DeleteFileActivity(long managementFileId, long activityId)
                 throw new BadRequestException("Activity with the given id does not match the management file id");
             }
 
-            if (!managementActivity.PropMgmtActivityStatusTypeCode.Equals(ManagementActivityStatusTypeCode.NOTSTARTED.ToString()))
-            {
-                throw new BadRequestException($"PropertyManagementActivity can not be deleted since it has already started");
-            }
-
-            var success = _managementActivityRepository.TryDeleteByFile(activityId, managementFileId);
-            _propertyRepository.CommitTransaction();
-
-            return success;
+            return DeleteActivity(activityId);
```

Comment:
re-used the same method now. Reduces code duplication and enforces same rules for all activity deletions